### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by keepitsimple**
Work in progress. Your feedback is highly appreciated.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
   "name": "@anticrm/platform-root",
   "version": "0.4.0",
+  "engines": {
```
Comment (haiodo) [inline package.json:4]: Root package json is for some useful operations, it will not have any respect to main rush reactor.

Comment (keepitsimple) [reply package.json:4]: I totally agree with your point. It's just gentle reminder to users who will try it that the project requires Node.js 14.17+. Some modern IDEs already respect it and notify a developer about mismatch. 
See https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines

It's a minor thing. I can remove it

Comment (haiodo) [reply package.json:4]: No need to remove, if you interested please check if we could add same check for rush.

Comment (keepitsimple) [reply package.json:4]: > No need to remove, if you interested please check if we could add same check for rush.

Ok. I will

File: README.md [hunk 2]
```diff
@@ -56,37 +139,145 @@ cd ./server/server
...
+│  └─ workspaces
+└─ tests - contains automated tests for the Platform code
+   └─ server
+```
+
+ 
+### How to run dev server
 
-### Running Dev server with auth/workspace
+> Before, please make sure that MongoDB & ElasticSearch are started and available.
```
Comment (BykhovDenis) [inline README.md:213]: Currently we don't need run ElasticSearch

File: README.md [hunk 3]
```diff
@@ -56,37 +139,145 @@ cd ./server/server
...
+│  ├─ services
+│  ├─ tools
+│  ├─ workspace
+│  └─ workspaces
+└─ tests - contains automated tests for the Platform code
+   └─ server
+```
+
+ 
+### How to run dev server
```
Comment (BykhovDenis) [inline README.md:211]: You can add information about run dev(demo) site without server, auth and any data storage(use memDB).  just `cd ./dev/prod` and `rushx dev`

File: README.md (hunk 2 above)
Comment (keepitsimple) [reply README.md:213]: ok

Comment (keepitsimple) [reply README.md:213]: @haiodo Can you clarify please when we need to run ElasticSearch?

File: README.md [hunk 4]
```diff
@@ -1,31 +1,115 @@
...
-
-[![Maintainability](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/maintainability)](https://codeclimate.com/github/hardcoreeng/platform/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/test_coverage)](https://codeclimate.com/github/hardcoreeng/platform/test_coverage)
+# Anticrm Platform [![Maintainability](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/maintainability)](https://codeclimate.com/github/hardcoreeng/platform/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/test_coverage)](https://codeclimate.com/github/hardcoreeng/platform/test_coverage)
+
+Business Applications Reimagined. Real time. Web first. Open source.
+
+
+**Description:** 
+
+The PLATFORM is an attempt to reimagine a concept of web platforms for business needs like WordPress, Bitrix, etc.
```
Comment (haiodo) [inline README.md:8]: I do not think we need this paragraph at this particular moment.

File: README.md [hunk 5]
```diff
@@ -56,37 +140,153 @@ cd ./server/server
...
+├─ CONTRIBUTING.md
+├─ package.json
+├─ rush.json
+├─ common - contains general configurtations for the project
+│  ├─ changes
+│  ├─ config
+│  ├─ git-hooks
+│  └─ scripts
+├─ dev - contains local development server
+│  ├─ platform-rig
```
Comment (haiodo) [inline README.md:165]: No inner listings please.

File: README.md [hunk 6]
```diff
@@ -56,37 +140,153 @@ cd ./server/server
...
+│  ├─ prod - frontend part of the project
+│  ├─ server - backend part of the project
+│  └─ ui-build
+├─ models - contains definitions for domain modals
+│  ├─ all
```
Comment (haiodo) [inline README.md:170]: No need for inner listings please.

File: README.md [hunk 7]
```diff
@@ -56,37 +140,153 @@ cd ./server/server
...
+│  ├─ chunter
+│  ├─ core
+│  ├─ dev
+│  ├─ fsm
+│  ├─ meeting
+│  ├─ recruting
+│  ├─ task
+│  └─ workbench
+├─ packages - contains code shared among client and server, except packages/platform which defines plugin architecture and client-side only. 
+│  ├─ core
```
Comment (haiodo) [inline README.md:181]: No need for inner listings please.

File: README.md [hunk 8]
```diff
@@ -56,37 +140,153 @@ cd ./server/server
...
+│  ├─ text
+│  ├─ theme
+│  ├─ ui
+│  └─ webrtco
+├─ plugins - contains client-side code packaged in form of Platform Plugins.
+│  └─ ...
+├─ products - contains examples of business aplications
+│  └─ hr-build
+├─ server - contains server-side code of the Platform
+│  ├─ accounts
```
Comment (haiodo) [inline README.md:198]: No need for inner listings please.

File: README.md [hunk 9]
```diff
@@ -56,37 +140,153 @@ cd ./server/server
...
+
+#### Force all parts of the project to be rebuilt
+```bash
+rush rebuild
+```
+
+#### Format all changed files
+
+```bash
+rushx format-diff
```
Comment (haiodo) [inline README.md:261]:
```suggestion
rush format
```

File: README.md [hunk 10]
```diff
@@ -56,37 +140,153 @@ cd ./server/server
...
+```
+
+#### Lint all changed files
+
+```bash
+rushx lint-diff
```
Comment (haiodo) [inline README.md:267]:
```suggestion
rush lint
```

File: README.md [hunk 11]
```diff
@@ -56,37 +140,153 @@ cd ./server/server
...
+```
+
+#### Execute tests on modified projects
+
+```bash
+rushx test-diff
```
Comment (haiodo) [inline README.md:273]:
```suggestion
rush test
```

File: README.md [hunk 12]
```diff
@@ -56,37 +140,153 @@ cd ./server/server
...
 ```
 
-## Dev tasks
+## Known issues
```
Comment (haiodo) [inline README.md:276]: No need for this section

File: README.md [hunk 13]
```diff
@@ -1,31 +1,115 @@
-# Anticrm Platform
-
-[![Maintainability](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/maintainability)](https://codeclimate.com/github/hardcoreeng/platform/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/test_coverage)](https://codeclimate.com/github/hardcoreeng/platform/test_coverage)
+# Anticrm Platform [![Maintainability](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/maintainability)](https://codeclimate.com/github/hardcoreeng/platform/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/test_coverage)](https://codeclimate.com/github/hardcoreeng/platform/test_coverage)
+
+Business Applications Reimagined. Real time. Web first. Open source.
```
Comment (haiodo) [inline README.md:3]: No need this line at the moment.

Comment (keepitsimple) [reply README.md:3]: fixed

File: README.md (hunk 4 above)
Comment (keepitsimple) [reply README.md:8]: fixed

File: README.md (hunk 5 above)
Comment (keepitsimple) [reply README.md:165]: fixed

File: README.md (hunk 6 above)
Comment (keepitsimple) [reply README.md:170]: fixed

File: README.md (hunk 7 above)
Comment (keepitsimple) [reply README.md:181]: fixed

File: README.md (hunk 8 above)
Comment (keepitsimple) [reply README.md:198]: fixed

File: README.md (hunk 9 above)
Comment (keepitsimple) [reply README.md:261]: fixed

File: README.md (hunk 10 above)
Comment (keepitsimple) [reply README.md:267]: fixed

File: README.md (hunk 11 above)
Comment (keepitsimple) [reply README.md:273]: fixed

File: README.md (hunk 12 above)
Comment (keepitsimple) [reply README.md:276]: Renamed to `Troubleshooting` 
It's important to have a section when developers/users can find answer on common issue. for example how to run project on M1 etc.

File: README.md [hunk 14]
```diff
@@ -56,37 +138,118 @@ cd ./server/server
...
 
-## Dev tasks
+## Troubleshooting
+This section covers common issues that developers may encounter when executing this project.
 
-All dev required tasks could be execured from root of project using rushx tool.
+### MacOS on M1
 
-### Format all changed files
+If you have MacOS on M1 it needs to use Node.js 16.x for the project.
```
Comment (haiodo) [inline README.md:243]: Better to put this in section about node jabs requirements, in this case it will be no troubleshooting required.

**Pull request closed without merging**